x-pack/legacy/plugins/maps/public/layers/tooltips/es_tooltip_property.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+import { ITooltipProperty } from './tooltip_property';
+import { IField } from '../fields/field';
+import { esFilters, IFieldType, IndexPattern } from '../../../../../../../src/plugins/data/public';
+
+export class ESTooltipProperty implements ITooltipProperty {
+  private _tooltipProperty: ITooltipProperty;
```

Comment:
flag as `readonly`.

this is not due to this PR in particular. Created https://github.com/elastic/kibana/issues/59695 to track this.